src/AccordionContainer/AccordionContainer.tsx

```diff
@@ -80,8 +80,8 @@ export class Provider extends React.Component<ProviderProps, ProviderState> {
                     }". Uuid property must be unique. See: https://github.com/springload/react-accessible-accordion#accordionitem`,
                 );
             }
-            if (this.props.accordion && newItem.expanded) {
-                // If this is a true accordion and the new item is expanded, then the others must be closed.
+            if (!this.props.allowMultipleExpanded && newItem.expanded) {
+                // If this is a true accordion that doesn't allow multiple expansions and the new item is expanded, then the others must be closed.
```

Comment:
This comment is probably redundant now, seeing as the name of the prop `allowMultipleExpanded` is kinda self-documenting, and the concept of a 'true accordion' is kinda ambiguous and confusing.